.github/workflows/workflow.yml

```diff
@@ -10,6 +10,7 @@ concurrency:
 
 env:
   NO_VENV: true
+  LATEST_VERSION: 23.9.3.12
```

Comment:
Should be replaced with `latest` when https://github.com/yandex/ch-backup/issues/84 get fixed.